hq/app/services/CacheService.scala

```diff
@@ -14,18 +16,59 @@ import scala.concurrent.duration._
 
 
 class CacheService(config: Configuration, lifecycle: ApplicationLifecycle, environment: Environment)(implicit ec: ExecutionContext) {
-  private val sgsBox: Box[Map[AwsAccount, Either[FailedAttempt, List[(SGOpenPortsDetail, Set[SGInUse])]]]] = Box(Map.empty)
   private val accounts = Config.getAwsAccounts(config)
+  private val startingCache = accounts.map(acc => (acc, Left(Failure.cacheServiceError(acc.id, "cache").attempt))).toMap
```

Comment:
Discussed using type aliases for the caches to remove some duplication. For example:
`type Tmp[Report] = Map[AwsAccount, Either[FailedAttempt, Report]]`

Having tried it, I have a minor preference for the original. The lines are longer this way, but scanning along a line to understand a val is easier to me than switching to reading 3-5 above then back.